tests/Microsoft.Identity.Web.Test/TokenAcquisitionAddInTests.cs

```diff
@@ -60,5 +61,51 @@ public async Task InvokeOnBeforeTokenAcquisitionForApp_InvokesEvent()
             Assert.NotNull(result);
             Assert.Equal(TokenSource.IdentityProvider, result.AuthenticationResultMetadata.TokenSource);
         }
+
+        [Fact]
+        public async Task InvokeOnBeforeTokenAcquisitionForUsernamePassword_InvokesEvent()
```

Comment:
Not sure what this is testing?
Isn't it possible to test the TokenAcquirer feature?
Here this is testing MSAL?

Reply:
Updated the test to add body parameter through extensibility and validate it through the mock handler.